core/point/point.go

```diff
@@ -46,55 +43,46 @@ func New(x, y int64) *Point {
 	}
 }
 
-// X returns the x-value.
+// X method receives a pointer argument of type *Point,
```

Comment:
This is a little wordy. The types gives much of the information. Just something like the following would be sufficient:

`// X returns the x-value.`